File: String/codeforces_281A.py

# A single line contains a non - empty word.This word consists of lowercase and uppercase English letters.The length of
# the word will not exceed 103.
# Output the given word after capitalization.

word = input()

# Capitalize the first letter of the word
modified_word = word[0].upper() + word[1:]

# Output the modified word
print(modified_word)

File: String/codeforces_281A.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/281/A
// submission link ===>> https://codeforces.com/contest/281/submission/209301918

#include <iostream>
#include <string>
using namespace std;

int main()
{
    string word;
    cin >> word;

    // Capitalize the first letter of the word
    word[0] = toupper(word[0]);

    // Output the modified word
    cout << word << endl;

    return 0;
}
